src/components/App.tsx

```diff
@@ -1,9 +1,92 @@
-import React from 'react'
+// import React from 'react'
+
+// export default function App() {
+//   return (
+//     <div className="reactApp">
+//         HELLO React
+//     </div>
+//   )
+// }
+import React from 'react';
```

Comment:
you should use 
```tsx
import * as React from "react";
```
not
```tsx
import React from "react";
```

Reply:
please read this tweet https://twitter.com/sebmarkbage/status/1250284377138802689?ref_src=twsrc%5Etfw%7Ctwcamp%5Etweetembed%7Ctwterm%5E1250284377138802689&ref_url=https%3A%2F%2Fshinyaigeek.dev%2Fpost%2Flog-ca-web-speed-hackathon